addons/vote.py

```diff
@@ -5,37 +5,117 @@
 import os
 
 class Vote:
+    pollcfg = {}
     vote_list = {}
-    cleanup_msgs_after = 30
+    poll_ongoing = False
 
     def __init__(self, bot):
         self.bot = bot
-        if os.path.isfile("votes.json"):
-            with open("votes.json", "r") as votefile:
-                self.vote_list = json.load(votefile)
+        if os.path.isfile("poll.json"):
+            with open("poll.json", "r") as pollfile:
+                self.pollcfg = json.load(pollfile)
+                if len(self.pollcfg):
+                    self.poll_ongoing = True
+                    vote_name = self.pollcfg["name"] + "_votes.json"
+                    if os.path.isfile(vote_name):
+                        with open(vote_name, "r") as votefile:
+                            self.vote_list = json.load(votefile)
+        else:
+            with open("poll.json", "w") as pollfile:
+                json.dump(self.pollcfg, pollfile)
         self.queue = asyncio.Queue()
+        
+    async def is_poll_ongoing(ctx):
+        if ctx.cog.poll_ongoing:
+            return True
+        else:
+            await ctx.send("There's no poll running!")
```

Comment:
This should be done in an error handler.

Reply:
It'll be a pain to handle this via error handling, especially since it's a check